CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+
+If no existing item describes your issue/feature, great - please file a new issue:
+
+### File a new Issue
+
+* Don't know whether you're reporting an issue or requesting a feature? File an issue
+* Have a question that you don't see answered in docs, videos, etc.? File an issue
+* Want to know if we're planning on building a particular feature? File an issue
+* Got a great idea for a new utility or feature? File an issue/request/idea
+* Don't understand how to do something? File an issue/Community Guidance Request
+* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+
+### Complete the template
+
+**Please include as much information as possible in your issue**. The more information you provide, the more likely your issue/ask will be understood and implemented. Helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
+* What build of Windows your device is running
+
+  👉 Tip: Run the following in PowerShell Core
+
+  ```powershell
+  C:\> $PSVersionTable.OS
+  Microsoft Windows 10.0.18909
+  ```
+
+  ... or in Windows PowerShell
+
+  ```powershell
+  C:\> $PSVersionTable.BuildVersion
+
+  Major  Minor  Build  Revision
+  -----  -----  -----  --------
+  10     0      18912  1001
+  ```
+
+  ... or Cmd:
+
+  ```cmd
+  C:\> ver
+
+  Microsoft Windows [Version 10.0.18900.1001]
+  ```
+
+* What tools and apps you're using (e.g. VS 2019, VSCode, etc.)
+* Don't assume we're experts in setting up YOUR environment and don't assume we are experts in YOUR workflow. Teach us to help you!
+* **We LOVE detailed repro steps!** What steps do we need to take to reproduce the issue? Assume we love to read repro steps. As much detail as you can stand is probably _barely_ enough detail for us!
+* Prefer error message text where possible or screenshots of errors if text cannot be captured
+* **If you intend to implement the fix/feature yourself then say so!** If you do not indicate otherwise we will assume that the issue is our to solve, or may label the issue as `Help-Wanted`.
+
+### DO NOT post "+1" comments
+
+> ⚠ DO NOT post "+1", "me too", or similar comments - they just add noise to an issue.
+
+If you don't have any additional info/context to add but would like to indicate that you're affected by the issue, upvote the original issue by clicking its [+😊] button and hitting 👍 (+1) icon. This way we can actually measure how impactful an issue is.
+
+---
+
+## Development
+
+### Fork, Clone, Branch and Create your PR
+
+Once you've discussed your proposed feature/fix/etc. with a team member, and you've agreed an approach or a spec has been written and approved, it's time to start development:
+
+1. Fork the repo if you haven't already
+1. Clone your fork locally
+1. Create & push a feature branch
+1. Create a [Draft Pull Request (PR)](https://github.blog/2019-02-14-introducing-draft-pull-requests/)

```

Comment:
We don't generally use draft PRs - why this guidance?

Reply:
Changed to just a Pull Request